controllers/v1/learning-resources.js

```diff
@@ -0,0 +1,373 @@
+                  "resourceType": "Learn",
+                  "node_id": 228
+              }
+          ]
+      }
+  }
+    **/
+
+    /**
+     * To get list of resources
+     * @method
+     * @name list
+     * @param  {req}  - requested data.
+     * @returns {json} Response consists list of learning resource
+    */
+
+    list(req) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let response = await learningResourceshelper.list(
+                    req.userDetails.userToken,
+                    req.pageSize,
+                    req.pageNo,
+                    req.body.filters ? req.body.filters : {},
+                    req.query.sortBy ? req.query.sortBy : ""
+                );
+
+                return resolve({ result: response.data, message: response.message });
+
+            } catch (error) {
+
+                return reject({
+                    status: error.status || httpStatusCode.internal_server_error.status,
+                    message: error.message || httpStatusCode.internal_server_error.message,
+                    errorObject: error
+                });
+
+            }
+        });
+    }
+    /**
+  * @api {get} /kendra/api/v1/learning-resources/getFilters
+  * To get learning resources filters
+  * @apiVersion 1.0.0
+  * @apiGroup Learning Resources
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /kendra/api/v1/learning-resources/getFilters
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  {
+    "message": "Learning resource form fetched successfully.",
+    "status": 200,
+    "result": [
+        {
+            "name": "All",
+            "icon": "documents-outline",
+            "value": []
+        },
+        {
+            "name": "Collections",
+            "icon": "documents-outline",
+            "value": [
+                "application/vnd.ekstep.content-collection"
+            ]
+        },
+        {
+            "name": "Documents",
+            "icon": "document-text-outline",
+            "value": [
+                "application/pdf",
+                "application/epub"
+            ]
+        },
+        {
+            "name": "video",
+            "icon": "play-circle-outline",
+            "value": [
+                "video/mp4",
+                "video/x-youtube",
+                "video/webm"
+            ]
+        },
+        {
+            "name": "interactive",
+            "icon": "play-circle-outline",
+            "value": [
+                "application/vnd.ekstep.ecml-archive",
+                "application/vnd.ekstep.h5p-archive",
+                "application/vnd.ekstep.html-archive"
+            ]
+        }
+    ]
+}
+    /**
+   * To get learning resources filters
+   * @method
+   * @name getFilters
+   * @param  {req}  - requested data.
+   * @returns {json} Response consists of learning resource filters
+  */
+
+    getFilters(req) {
```

Comment:
@aman-tunerlabs - Rename it to just filters please, get will be there in method type

Reply:
@rakeshSgr  Please let first frontend team know about the changes as they are using above api and make the required changes.